wind_tribe_tower.logic

```diff
@@ -0,0 +1,5 @@
+area WindTribeTower:
+  room GF
+  room 1F
+  room 2F
+  room 3F
```

Comment:
I think we required names to start with either a capital letter or be quoted? I.e.
```suggestion
  room "1F"
  room "2F"
  room "3F"
```
(At least, that's how the reference parser I implemented works atm)